docs/standard/native-interop/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Native interoperability
+description: Learn how to interface with native components in .NET.
+author: blackdwarf
+ms.author: ronpet
+ms.date: 06/20/2016
+ms.technology: dotnet-standard
```

Comment:
Do any of these attributes need to be updated?

Reply:
yes please. author should be the GitHub alias of the person writing the topic. 
ms.date should be today's date. remove ms.technology.
ms.author is the alias of the Microsoft's person responsible for keeping it up-to-date.